requirements-dev.txt

```
@@ -1,5 +1,5 @@
 pip~=22.0
-git+https://github.com/PennyLaneAI/pennylane.git@master
+git+https://github.com/PennyLaneAI/pennylane.git@inner-execute
```

Change this once https://github.com/PennyLaneAI/pennylane/pull/5791 is merged.

```suggestion
git+https://github.com/PennyLaneAI/pennylane.git@master
```